### PR TITLE
Milestone 1: Initialize project with components

### PR DESCRIPTION
**Pull request by zhadier**
# The following changes were implemented in this Milestone :rice_cracker: 
- Initialise React app and added React Redux
- Structured application files using a "feature folder" approach and used the ducks pattern for Redux files.
- Added React Router and set up two Routes and Links for the app's navigation to Books and Categories.
### Books Section
- Displays the list of books with the Remove button.
- Has a form for adding a book (no functionality yet)
 ### Categories Section
-  Displays "Under construction" text only.

## Thanks for taking out the time to review this project 🧨
It really is appreciated :) ❤️ 
- Note: I personally love optional suggestions so please do leave some 😄

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const Book = () => (
```
Comment (Meltrust) [inline src/components/Book.js]: - [ ] Please, make sure this "Book" component takes the book data as a _prop_ and renders the title and the author. 

## Explanation and hints:

### Example of how to create the new `Book` component that takes the book data as props:

```JAVASCRIPT
const Book = (props) => {
  const {
    id, genre, title, author, currentChapter,
  } = props;
```

That way, your components will communicate with each other, ready for the data, and for the next steps! 💪 

I hope this is helpful.

_These are just examples of one way to do it based on my own experience and effort. Included them here just to help and give you an idea, and should not be taken as an "official" way to do it. For this milestone, you only need the title and the author._

File: src/components/Booklist.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import Book from './Book';
+import AddForm from './AddForm';
+
+const Booklist = () => (
```
Comment (Meltrust) [inline src/components/Booklist.js]: - [ ] The books should be rendered by the "BookLIst" component, using the new "Book" component from the previous step for the book data (props).

## Explanation and hints:

### On your BooksList main component, just use `map` to pass all the props to every `Book` and render them like this example:

```JAVASCRIPT
import Book from '../Book.js';

const BookList = () => {

const booksData = [ // added books here just to ilustrate
    { id: 1, title: 'The Hobbit', author: 'J.R.R. Tolkien' },
    { id: 2, title: 'The Da Vinci Code', author: 'Dan Brown' },
    { id: 3, title: 'Pinocchio', author: 'Carlo Collodi' },
  ];

const bookItems = booksData.map((element) => (
    <Book
      key={element.id}
      id={element.id}
      genre={element.genre}
      title={element.title}
      author={element.author}
      percentageCompleted={element.percentageCompleted}
      currentChapter={element.currentChapter}
    />
  ));

  return (
    <>
      {bookItems}
      <Form />
    </>
  );
}

export default BookList;
```

I hope this is helpful.
That way, your project will be ready for the data and for the next step!! 💪 

_These are just examples of one way to do it based on my own experience and effort. Included them here just to help and give you an idea, and should not be taken as an "official" way to do it. For this milestone, you only need the title and the author._

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @zhadier,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Nice code organization✔️
- Cool pull request ✔️  
- The project works well ✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.js (hunk 1 above)
Comment (zhadier) [reply src/components/Book.js]: Thank you so much for your guidance, this cleared a lot of doubts.

File: src/components/Booklist.js (hunk 2 above)
Comment (zhadier) [reply src/components/Booklist.js]: This was very helpful, thank you so much

**Pull request merged**